openmdao/core/problem.py

```diff
@@ -70,6 +82,9 @@ class Problem(object):
         0 -- Newly initialized problem or newly added model.
         1 -- The `setup` method has been called, but vectors not initialized.
         2 -- The `final_setup` has been run, everything ready to run.
+    cite: str
```

Comment:
`cite: str` should be `cite : str`; listing should be capitalized; `citataions` should be `citations`; `class` should be followed by a period